app/main.py

```diff
@@ -1,3 +1,67 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, number):
+        if isinstance(number, Distance):
+            return Distance(self.km + number.km)
+        elif isinstance(number, (int, float)):
```

Comment:
here and in all magic methods below u don't have to check wether second arg is int or float. Task was to ensure that second arg is Distance instance.